sunshine/process.cpp

```diff
@@ -147,6 +147,8 @@ int proc_t::running() {
 
 void proc_t::terminate() {
   std::error_code ec;
+  int app_id = _app_id;
+ 
```

Comment:
why not just use `_app_id` instead of creating a new variable?

Reply:
as on line 156 _app_id is set to -1 this was to prevent any side effects from original behaviour